src/builder.rs

```diff
@@ -95,123 +95,6 @@ impl TransactionBuilder {
     }
 }
 
-/// Builds a ReissueRequest from a ReissueTransaction and
-/// any number of (input) DBC hashes with associated ownership share(s).
-#[derive(Debug, Default)]
-pub struct ReissueRequestBuilder {
```

Comment:
It seems to me that the client will still need a similar builder for aggregating signature shares and generating the spendbook entry, no?

Reply:
yes.. maybe that should be part of blsttc? a more general purpose Signature Share Aggregator?

Reply:
That would be nice indeed @iancoleman ;-)